pkg/infra/aliyun/ali_ecs.go

```diff
@@ -445,7 +445,12 @@ func (a *AliProvider) RunInstances(instanceRole string, count int) error {
 
 	request := ecs.CreateRunInstancesRequest()
 	request.Scheme = Scheme
-	request.ImageId = ImageID
+	system := a.Cluster.Spec.Masters.ImageID
+	if system == CentOS || system == "" {
+		request.ImageId = CentOSImageID
+	} else if system == Ubuntu {
```

Comment:
what happens in other situations? All situations should be considered at all times.